### DISCUSSION TITLE
Camera con motion detection e registrazione

### DISCUSSION THREAD

**Discussion by fily1212** [indomus/forum#1723] [Hardware & Firmware]
Ciao a tutti. 
Scrivo per chiedervi un consiglio per l'acquisto di una telecamera per interni per il sistema di videosorveglianza. Non ho ancora deciso se la telecamera dovrà essere wifi o ethernet, però vorrei portare un cavo soltanto, quindi se ethernet preferirei che avesse anche l'alimentazione inclusa. 
La domanda che volevo porvi è questa: ho il sospetto che convenga comprare una camera con motion detection integrata che triggeri home assistant invece che fare motion detection con MotionEye, per risparmiare CPU, in ottica di metterne anche più di una. E' corretto il sospetto ed esiste una camera che può fare questo? 
Mi piacerebbe fare una configurazione tale per cui, se la casa è armata ascolta questo sensore di movimento, se la casa non è armata invece lo ignora. Se questo sensore si attiva allora fa partire la registrazione e magari invia anche su telegram qualche foto o uno spezzone di video. E perché no, un allarme sonoro da alexa o in futuro da una sirena. 

Vedendo questa recensione sul sito di Indomus [Ankee c800](https://indomus.it/componenti/ankee-c800-turret-i91bm-telecamera-di-sicurezza-poe-ip-true-4k-ultra-hd/) mi sembra di capire che questa telecamera possa avere un motion detection. Però nell'articolo si parla anche di sensore virtuale con MotionEye. Dallo screen finale invece non riesco a capire se è il sensore della camera o il sensore di MotionEye, e quindi se questa soluzione potrebbe essere buona. 
Avete dei consigli di camere che potrebbero far al caso mio che si integrano bene in Home Assistant? 
grazie

**Comment by KC-inDomus** [comment]
la annke che citi ha il suo motore di riconoscimento, però è il suo se la integri su motioneye, configuri il motion detection di motioneye, fai prima (ed è più versatile)

**Comment by NdK73** [comment]
Con MotionEye sto collaudando le Esp32Cam.
La qualità del video è davvero bassa, ma il riconoscimento del movimento pare funzionare bene, a parte un po' di lag.
Alla fine costano poco più di un sensore PIR e il video è quasi "in omaggio" :) Può valer la pena di considerarle. Se poi invece si vuole risparmiare la CPU ci si può aggiungere un modulino PIR e bypassare MotionEye.
